packages/now-static-build/src/index.ts

```diff
@@ -28,6 +28,7 @@ import {
   PrepareCacheOptions,
 } from '@now/build-utils';
 import { Route, Source } from '@now/routing-utils';
+import { NowBuildError } from '@now/build-utils';
```

Comment:
Can you add this to line 28 so that there is only one reference of `@now/build-utils`?